source/ddox/ddoc.d

```diff
@@ -19,7 +19,7 @@ import std.uni : isAlpha;
 // TODO: support escapes section
 
 
-static this()
+shared static this()
```

Comment:
`s_standardMacros` would have to be made `__gshared` for this to work, but why not.